client/rest/src/test/java/org/elasticsearch/client/RequestTests.java

```diff
@@ -24,15 +24,16 @@
 import org.apache.http.entity.ByteArrayEntity;
 import org.apache.http.entity.ContentType;
 import org.apache.http.entity.StringEntity;
-import org.apache.http.nio.entity.NStringEntity;
-import org.elasticsearch.client.HttpAsyncResponseConsumerFactory.HeapBufferedResponseConsumerFactory;
 
 import java.io.ByteArrayOutputStream;
 import java.io.IOException;
-import java.util.ArrayList;
 import java.util.HashMap;
-import java.util.List;
 import java.util.Map;
+import org.apache.http.nio.entity.NStringEntity;
```

Comment:
I think this change is worse? Maybe revert it?

Reply:
yep sorry